src/test/resources/feature/TestScenarios.feature

```diff
@@ -0,0 +1,49 @@
+@Claudia
+Feature: Add, remove and modify items from shopping cart
```

Comment:
Could you please rename your TestScenarios.feature file to more specific (the area you are testing).

Reply:
Done